src/components/Header/index.jsx

```diff
@@ -36,6 +38,7 @@ class Header extends React.Component {
           </div>
           {email && <Dropdown
             title={email}
+            iconElement={<img src={avatarIcon} alt="Avatar icon for profile" />}
```

Comment:
The image used here should be the profile image from the LMS. However, I don't believe we currently have a way to retrieve the authenticated user's profile image as it doesn't come back in the JWT token and I am unaware if there's currently an endpoint in LMS where we can fetch the user's profile image.

@georgebabey Thoughts on how we can/should get the profile image url?

Reply:
@adamstankiewicz @georgebabey We can use `api/user/v1/accounts/{username}` endpoint to get  profile images but the problem is that we don't have `username` available in edx-portal. There is another `api/user/v1/me` endpoint which can return username of logged in user but to me it doesn't look right to make a request just to get the logged in user's username.

Can not we get username inside the response returned by oauth2 authentication?